src/taskgraph/main.py

```diff
@@ -652,6 +652,31 @@ def image_digest(args):
         sys.exit(1)
 
 
+@command(
+    "load-task",
+    help="Loads a pre-built Docker image and drops you into a container with "
+    "the same environment variables and run-task setup as the specified task. "
+    "The task's payload.command will be replaced with 'bash'. You need to have "
+    "docker installed and running for this to work.",
+)
+@argument(
+    "task_id",
+    help="Load the image at public/image.tar.zst in this task, "
+    "rather than searching the index",
```

Comment:
"searching the index" doesn't seem to be an option here, so maybe remove that part?

Reply:
Oops, this was copy/pasted from the `load-image` command and I neglected to update the description.